### PR TITLE
Add the TO RUN syntax in the language guide.

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

File: doc/language_guide.md [hunk 1]
```diff
@@ -442,6 +442,55 @@ INTO my_db.my_result_table;
     - [BARON](https://www.minlp.com/baron): used to solve the non-linear programming problems.
 - The `INTO my_db.my_result_table` means that the solved result would be saved in the `my_db.my_result_table` table.
 
+## TO RUN Syntax
+
+SQLFlow provide `TO RUN` statement to execute a runnable to execute complicated
```
Comment (sneaxiy) [inline doc/language_guide.md]: `provide` -> `provides`

`TO RUN statement` -> `the TO RUN statement`

`to execute a runnable to execute ...`. There are too many words `execute`. Try to use an alternative word.

File: doc/language_guide.md [hunk 2]
```diff
@@ -442,6 +442,55 @@ INTO my_db.my_result_table;
...
+data preprocessing and analysis. The runnables can be implemented using any
+program language such as Python, C++, Go .etc and is released as Docker images.
+The Syntax is as follows:
```
Comment (sneaxiy) [inline doc/language_guide.md]: `Syntax` -> `syntax`.

Comment (brightcoder01) [reply doc/language_guide.md]: Good catch, updated.

File: doc/language_guide.md (hunk 1 above)
Comment (brightcoder01) [reply doc/language_guide.md]: Good catch, updated

**Review by sneaxiy — APPROVED** [review]

**Pull request merged**